app/models/enrollment.rb

```diff
@@ -0,0 +1,10 @@
+class Enrollment < ActiveRecord::Base
```

Comment:
The intent of this model is to provide a glue between Games and Players right?

Assuming that is the case, I would recommend a model name modifying `Game`. Something like `GameEnrollment`, `GamePlayer`, `GameDerp`. Enrollments is somewhat ambiguous IMO